SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -19,12 +19,17 @@ DEFINE_LOG_CATEGORY(LogSpatialWorkerConnection);
 
 namespace
 {
-SpatialGDK::ComponentData ToComponentData(FWorkerComponentData* Data)
+SpatialGDK::ComponentData ToComponentData(const FWorkerComponentData* Data)
```

Comment:
I think while syntactically fine this is semantically incorrect. As we are converting ownership of the underlying data.